src/components/MenuPanel/MenuPanel.js

```diff
@@ -59,21 +60,15 @@ class MenuPanel extends React.Component {
   render() {
     const {
       apps,
-      activeAppId,
-      activeInstanceId,
       notificationsObservable,
-      onOpenApp,
       onClearAllNotifications,
       onOpenNotification,
     } = this.props
     const { notificationsOpened } = this.state
-    const menuApps = [
-      appHome,
-      ...addIcons(apps),
-      appPermissions,
-      appApps,
-      appSettings,
-    ]
+
+    const daoApps = addIcons(apps)
+    const menuApps = [appHome, daoApps, appPermissions, appApps, appSettings]
```

Comment:
It might be nice to move all this logic into a `renderMenuApps()` method so we don't have to use `Array.isArray()` to figure out what should be included in the loader.

Reply:
I’m not sure I understand how we could remove `Array.isArray()`? The idea is to be able to move the apps to the position we want in `menuApps`, and detecting them with `Array.isArray()`.